haystack/nodes/search_engine/web.py

```diff
@@ -1,7 +1,8 @@
 import pydoc
 from typing import List, Dict, Any, Optional, Union, Tuple, Type
 
-from haystack import BaseComponent, MultiLabel, Document
+from haystack.nodes.base import BaseComponent
+from haystack import MultiLabel, Document
```

Comment:
Just curious, how come that this was split? Are there circular deps?

Reply:
Yes!

For example, see this [log](https://github.com/deepset-ai/haystack/actions/runs/4594265828/jobs/8113085735).

Reply:
These imports are going to bite us back for real one day or another :smiling_face_with_tear: